calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -74,8 +74,16 @@ kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySync
 ### Enable WAF
 
 1. On the Manager UI, click **Threat Defense**, **Web Application Firewall**, **Configure Web Application Firewall**.
-2. For each service that you want to enable for WAF, click the **Action** column, select Enable.
+2. On the Select Services to Inspect page, choose all the services for which you want WAF to inspect the traffic.
```

Comment:
2. Select the services you want WAF to inspect, and then click **Confirm Selections**.
3. On the **Web Application Firewall** page, you can verify that WAF is enabled for a service by locating the service and checking that the **Status** column says **Enabled**.
4. To make further changes to a service, click **Actions**, and then **Enable** or **Disable**.

Reply:
It's fixed